eosio-web-ide.dockerfile

```diff
@@ -1,4 +1,4 @@
-# docker build -t eosio/eosio-web-ide:v0.1.0 -f eosio-web-ide.dockerfile .
+# docker build -t eosio/eosio-web-ide:v0.1.1 -f eosio-web-ide.dockerfile .
```

Comment:
Is it necessary / advisable to version lock the Docker container for this?  Should we remove the version so that the latest container gets used?